src/components/dialogs/network-modifications/two-windings-transformer/characteristics-pane/two-windings-transformer-characteristics-pane.js

```diff
@@ -114,13 +109,11 @@ const TwoWindingsTransformerCharacteristicsPane = ({
 
     return (
         <>
-            <Grid container spacing={2}>
-                <Grid item xs={12}>
-                    <h3 className={classes.h3}>
-                        <FormattedMessage id="Characteristics" />
-                    </h3>
-                </Grid>
-            </Grid>
+            <GridSection
+                title={'Characteristics'}
+                heading="3"
```

Comment:
it's already the default value, no need to specify it

Reply:
Thanks. Fixed.